src/app/shared/components/common/modal-dialog-link/modal-dialog-link.component.ts

```diff
@@ -8,12 +9,19 @@ import { ModalDialogComponent, ModalOptions } from 'ish-shared/components/common
  * Displays a link (see parameter linkText). If the user clicks the link a modal dialog opens displaying some information ( dynamic input content ).
  * The component is not designed or intended to contain any logic, but only to display text.
  *
+ * The dynamic input content could be loaded on initialization (default) or on demand with lazy loading. This can be set via the lazyLoading input parameter.
+ * If the content should be loaded on demand, then the input content must be wrapped within a ng-container, which uses the LazyLoadingContentDirective.
+ * The component has now access to the template reference and can lazy load the input content, when the modal is opened.
+ *
  * @example
  *<ish-modal-dialog-link
     linkText="checkout.tac.link"
     [options]="{ titleText: 'checkout.termsandconditions.details.title' | translate, size: 'lg' }"
+    [lazyLoading]="true"
   >
-    <ish-content-include includeId="include.dialog.privacyPolicy.pagelet2-Include"></ish-content-include>
+    <ng-container *ishLazyLoadingContent>
```

Comment:
Since you use a structural directive that expands into an `<ng-template>`, are you sure you need to wrap the lazy rendered content in an `<ng-container>`? 

Have you tried simply applying the structural directive to the content?